acra-censor/acra-censor_implementation.go

```diff
@@ -20,23 +20,26 @@ import (
 	"github.com/cossacklabs/acra/acra-censor/common"
 	"github.com/cossacklabs/acra/acra-censor/handlers"
 	log "github.com/sirupsen/logrus"
+	"os"
 )
 
 // ServiceName to use in logs
 const ServiceName = "acra-censor"
 
 // AcraCensor describes censor data: query handler, logger and reaction on parsing errors.
 type AcraCensor struct {
-	handlers         []QueryHandlerInterface
-	ignoreParseError bool
-	logger           *log.Entry
+	handlers           []QueryHandlerInterface
+	ignoreParseError   bool
+	parseErrorsLogPath string
+	logger             *log.Entry
 }
 
 // NewAcraCensor creates new censor object.
 func NewAcraCensor() *AcraCensor {
 	acraCensor := &AcraCensor{}
 	acraCensor.logger = log.WithField("service", ServiceName)
 	acraCensor.ignoreParseError = false
+	acraCensor.parseErrorsLogPath = ""
```

Comment:
we don't need such initialization because it's already initialized with [empty string](https://tour.golang.org/basics/12). Above assigning is redundant too